src/products/products_function.py

```diff
@@ -0,0 +1,82 @@
+import os 
+# Function to clear the console based on the operating system
+
+def clear_console():
+    os.system('clear')
+
+# This is the main menu 
+
+
```

Comment:
Worth noting here that we'll be moving data related to the functionality of the app into a different directory later when we start persisting data in csv file format. Also, I think a comment would be appropriate here to describe the purpose/role of the list.

Reply:
resolved